src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/IsInRangeConverter.shared.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.CommunityToolkit.Extensions;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	public class IsInRangeConverter : BindableObjectExtension, IValueConverter
```

Comment:
It should be inherited from "ValueConverterExtension"

Just make ValueConverterExtension inherited from BindableObject

```csharp
public abstract class ValueConverterExtension : BindableObject, IMarkupExtension<IValueConverter>
```

Reply:
Done